src/core/TriangleSplitter.js

```diff
@@ -7,13 +7,23 @@ import { isTriDegenerate } from './utils/triangleUtils.js';
 const EPSILON = 1e-10;
 const COPLANAR_EPSILON = 1e-10;
 const PARALLEL_EPSILON = 1e-10;
+const COPLANAR_EPSILON_MAX = 1e-4;
+//*/
+
+// XAVIER: increase epsilon values:
+/*const EPSILON = 1e-5;
+const COPLANAR_EPSILON = 1e-5;
+const PARALLEL_EPSILON = 1e-5; //*/
```

Comment:
These values look left over from some debugging?

Reply:
Yes it can be removed